test/text/vtt_text_parser_unit.js

```diff
@@ -827,11 +827,12 @@ describe('VttTextParser', () => {
               },
             ],
           },
-          {
-            startTime: 90,
-            endTime: 100,
-            payload: '<b>Test<i>8</b>',
-          },
+          // NOTE: This is isn't going to work with tXml currently
```

Comment:
I have cloned your branch and uncommented this case and it works correctly, please revert this change.